backend/molgenis-emx2-semantics/src/main/java/org/molgenis/emx2/semantics/rdf/ValueToRDF.java

```diff
@@ -135,6 +135,9 @@ public static Value applyFormatting(Object o, Column column, String schemaContex
 
     } else {
 
+      if (o == null) {
```

Comment:
I would either connect it to the others in the big if, else if.

But better would it to rewrite as a switch case. (which is performs faster)

Reply:
https://stackoverflow.com/questions/33051931/if-else-vs-switch-performance-in-java

tldr;

Switch perf is better than if else as in case of switch there will be one time evaluation . Once it evaluated the switch it knows which case needs to be executed but in case of if else it has to go through all conditions in case of worst scenario.

The longer the list condition, better will be switch performance but for shorter list (just two conditions), it can be slower also

Reply:
Great suggestion! This code is heavily used during RDF export so performance matters here. Now implemented as switch.